packages/botonic-core/src/router.js

```diff
@@ -149,34 +150,24 @@ export class Router {
     return undefined
   }
 
-  getRoute(input, routes, session, previousValue) {
+  getRoute(input, routes, session) {
     const computedRoutes = isFunction(routes)
       ? routes({ input, session })
       : routes
     /* Find the route that matches the given input, if it match with some of the entries,
       return the whole Route of the entry with optional params captured if matcher was a regex */
     let params = {}
-    let childRoute = undefined
-    let route = computedRoutes.find(r =>
+    const route = computedRoutes.find(r =>
       Object.entries(r)
         .filter(([key, {}]) => key != 'action' && key != 'childRoutes')
         .some(([key, value]) => {
-          let currentValue = value
-          if (previousValue) currentValue = previousValue.concat('/', value)
-          const match = this.matchRoute(r, key, currentValue, input, session)
-          if (!match && r.childRoutes && !childRoute) {
-            childRoute = this.getRoute(
-              input,
-              r.childRoutes,
-              session,
-              currentValue
-            )
-          }
```

Comment:
Are we losing something removing this code?

Reply:
I added this in the old PR because I thought it was necessary but it was not.